app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -133,7 +133,6 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
     }
 
     private lateinit var navigationToolbar: Toolbar
-
```

Comment:
nit: try not to remove unrelated lines :)

Reply:
my bad!